.github/workflows/run_main_benchmarks.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# This script is responsible for cloning the main repository from GitHub and running Java Microbenchmark Harness (JMH) benchmarks.
+
+# Change directory to the one named after the branch name. 
```

Comment:
All the comments from this one on in this script are unnecessary and can be deleted